media/js/devreg/submit.js

```diff
@@ -105,6 +105,8 @@
         return $('#id_free_platforms option:selected').length ||
             $('#id_paid_platforms option[value="paid-firefoxos"]:selected').length ||
             $('#id_paid_platforms option[value="paid-android-mobile"]:selected').length ||
+            $('#id_paid_platforms option[value="paid-android-tablet"]:selected').length ||
+            $('#id_paid_platforms option[value="paid-desktop"]:selected').length ||
```

Comment:
Actually is there a condition where we want to hide the packaged app tab any more? Thinking I can just remove this whole function.